tests/embedding_net_test.py

```diff
@@ -454,6 +459,10 @@ def test_lru_block_isolated(
     sequence_len: int = 50,
 ):
     """Run some random data through an LRUBlock."""
+
+    if sys.version_info >= (3, 13) and mode == "scan":
+        pytest.xfail("torch.compiler is not yet supported on Python >= 3.13")
+
```

Comment:
I think it would more transparent to add this as a param case above: 
```
@pytest.mark.parametrize("mode", ["loop", pytest.param("scan", marks=pytest.mark.xfail(reason="...", strict=True)], ids=["loop", "scan"])
```
and set `strict=True` to let it fail once `python 3.13` is fixed. 

same for the other tests above, and below.